### ISSUE TITLE
DEPRECATION: Overriding init without calling this._super is deprecated

### ISSUE THREAD

**Issue by adamreisnz** [jkarsrud/ember-cli-defeatureify#19]
```
DEPRECATION: Overriding init without calling this._super is deprecated. 
  Please call `this._super.init && this._super.init.apply(this, arguments);` addon: `ember-cli-defeatureify`
```

Version 1.1.1 and `ember-cli` version 2.6.2.

**Comment by adamreisnz** [comment]
Is anyone still maintaining this package?

@nathanhammond I see you've forked this and did some more work on it here https://github.com/nathanhammond/ember-cli-defeatureify, are you able to merge those changes in and fix the deprecation warnings?

**Comment by dwickern** [comment]
I've been staring at this deprecation for a while now. It's a trivial fix. I'd be happy to open a PR if there's someone around who can merge it and make a release.

**Comment by nathanhammond** [comment]
@dwickern Would you be willing to update the Ember CLI version for this addon? I have commit privs but not publish. Part of the way there is better than nothing. :P

**Comment by jkarsrud** [comment]
A PR is most welcome @dwickern!

I added you as a collaborator on npm now, @nathanhammond, so I guess you should be able to publish now :)

**Comment by nathanhammond** [comment]
@jkarsrud Why, thank you! Will take good care of things, promise. ❤️

**Comment by nathanhammond** [comment]
v1.1.2 published.

**Comment by adamreisnz** [comment]
Awesome guys, thanks for looking into it.

**Comment by adamreisnz** [comment]
@Actually still getting a deprecation warning:

DEPRECATION: The `app-prefix` hook used in ember-cli-defeatureify is deprecated. The addon should generate a module and have consumers `require` it.

Are you able to address that one as well? I can raise a separate issue for it.

**Comment by nathanhammond** [comment]
@adambuczynski That's https://github.com/jkarsrud/ember-cli-defeatureify/pull/16 ... but it depends on https://github.com/ember-cli/ember-cli/pull/5582 which is blocked on spooky action at a distance in the Ember CLI test suite that I've not been able to figure out in spite of 40+ hours of research.

**Comment by adamreisnz** [comment]
I see, yes, thanks. I'll ignore that message for the mean time then. 👍
